plugin-veracode-sca/src/main/java/com/tracelink/prodsec/plugin/veracode/sca/controller/VeracodeScaConfigurationsController.java

```diff
@@ -81,7 +81,7 @@ public SynapseModelAndView getConfigurations() {
 	@PostMapping("client")
 	public String setApiClient(@RequestParam String apiId, @RequestParam String apiSecretKey,
 			RedirectAttributes redirectAttributes) {
-		if (apiId.equals("") || apiSecretKey.equals("")) {
+		if ("".equals(apiId) || "".equals(apiSecretKey)) {
```

Comment:
There's a lot of these in here. We should probably just go back and convert all of these to StringUtils.isBlank